integration-cli/docker_cli_ps_test.go

```diff
@@ -554,3 +557,23 @@ func (s *DockerSuite) TestPsFormatHeaders(c *check.C) {
 		c.Fatalf(`Expected 'NAMES\ntest\n', got %v`, out)
 	}
 }
+
+func (s *DockerSuite) TestPsDefaultFormatAndQuiet(c *check.C) {
+	config := `{
+		"psFormat": "{{ .ID }} default"
+}`
+	d, err := ioutil.TempDir("", "integration-cli-")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(d)
+
+	err = ioutil.WriteFile(filepath.Join(d, "config.json"), []byte(config), 0644)
```

Comment:
"precedence".... meaning it would override "-q" ?  hmmmm  I missed that.  I wonder if that's correct.
Should `--format` be a way to override the default (either from config file or true default) or should it also turn off `-q`?
I tend to think of situations where people specify flags because they're really really anal and want to make sure that they get exactly what they want w/o fear of some default changing on them.  I'm leaning towards `--format` falling into that category.

For example, should:
`docker ps --format table -q`
and
`docker ps -q`
really do different things when the default for "format" is "table" ?  I'm not sure specifying a default should change the result.  Thoughts?